README.md

```diff
@@ -170,35 +113,105 @@ More information about the
 [audit service](https://wiki.duraspace.org/display/FF/Design+-+Audit+Service)
 is available on the Fedora wiki.
 
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| audit.enabled | Enables/disables audit triplestore service  | false |
+| audit.input.stream | Audit Service jms message stream | broker:topic:fedora |
+| audit.event.baseUri | The baseUri to use for event URIs in the triplestore. A `UUID` will be appended to this value, forming, for instance: `http://example.com/event/{UUID}` | http://example.com/event |
+| audit.triplestore.baseUrl| The base url for the external triplestore service | http://localhost:3030/fuseki/test/update |
+| audit.triplestore.authUsername| Username for basic authentication against triplestore | |
+| audit.triplestore.authPassword| Password for basic authentication against triplestore | |
+| audit.filter.containers |  A comma-delimited list of URIs to be filtered (ignored) by the audit service | http://localhost:8080/fcrepo/rest/audit | 
+
+
 ### Repository Indexer (Solr)
 
 This application listens to Fedora's event stream and
 indexes objects into an external Solr server.
 
-### Repository Indexer (Triplestore) (not currently available)
+#### Properties
+| Name      | Description| Default Value |
+| :---      | :---| :----   |
+| solr.indexer.enabled | Enables/disables the SOLR indexing service. Disabled by default | false | 
+| error.maxRedeliveries |       | 10 |
+| fcrepo.checkHasIndexingTransformation |       | true |
+| fcrepo.defaultTransform |   ?    | null |
+| input.stream |   The JMS topic or queue serving as the message source    | broker:topic:fedora |
+| solr.reindex.stream |   The JMS topic or queue serving as the reindex message source    | broker:queue:solr.reindex |
+| solr.commitWithin |   Milliseconds within which commits should occur    | 10000 |
+| indexing.predicate |   ?    | false |
+| ldpath.service.baseUrl |   The LDPath service base url    | http://localhost:9085/ldpath |
+| filter.containers |   A comma-separate list of containers that should be ignored by the indexer  | http://localhost:8080/fcrepo/rest/audit |
```

Comment:
Same here, does this default value still make sense?

Reply:
same.